ui/src/components/Form/BorderButton/index.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames/bind';
+import React from 'react';
+import { MdKeyboardArrowDown } from 'react-icons/md';
+
+import style from './style.module.scss';
+
+const c = classNames.bind(style);
+
+export interface ButtonProps {
+  /**
+   * Rander button content
+   */
+  children: React.ReactElement;
+  /**
+   * If user want to show down icon
+   */
+  hasNoUpArrow?: boolean;
+}
+
+/**
+ * A BorderButton is a container wher user can rander image or text with dropdown icon.
+ */
+
+const BorderButton: React.FC<ButtonProps> = (p) => {
```

Comment:
1. Let's not leave an empty line b/w function and its description comment
2. Typo in description. `rander` should be `render`